docs/2021-isc21/practical_info.md

```diff
@@ -11,6 +11,14 @@ Tutorial attendees will be able to log in to a prepared environment
 where they can follow along with hands-on examples, or solve the
 tutorial exercises.
 
+You can create an account for the prepared environment by filling
+out the [account form](https://mokey.isc21.learnhpc.eu/auth/signup).
+The accounts will _not_ be approved until the day before the event, so 
+**please remember to keep a record of your username and password**.
+
+Once the account is approved, you will be able to access the system
+at https://isc21.learnhpc.eu/ (or via `ssh` to `isc21.learnhpc.eu`).
```

Comment:
Make this a clickable link?

`[https://isc21.learnhpc.eu/](https://isc21.learnhpc.eu/)`

Reply:
That doesn't happen automatically? Clearly mkdocs is rubbish

Reply:
Done in https://github.com/easybuilders/easybuild-tutorial/pull/60/commits/df15db3668c37c890d4f62c2b3f6f49259aa5620